frame/staking/src/pallet/mod.rs

```diff
@@ -802,7 +806,7 @@ pub mod pallet {
 			<Payee<T>>::insert(&stash, payee);
 
 			let current_era = CurrentEra::<T>::get().unwrap_or(0);
-			let history_depth = Self::history_depth();
+			let history_depth = T::HistoryDepth::get();
```

Comment:
This would mean that we should re-benchmark, and expect one less read now with this PR.